src/views/List.jsx

```diff
@@ -1,15 +1,45 @@
 import { ListItem } from '../components';
+import { useState } from 'react';
 
 export function List({ data }) {
+	const [searchItem, setSearchItem] = useState('');
+
+	const handleTextChange = (event) => {
```

Comment:
There may be an opportunity here to make this function even more concise by using an implicit return!

Reply:
Thank you for your thoughtful suggestion, will definitely address this.